cosmoz-bottom-bar.js

```diff
@@ -213,37 +213,33 @@ class CosmozBottomBar extends PolymerElement {
 
 	_childrenUpdated(info) {
 		const addedNodes = info.addedNodes.filter(this._isActionNode),
-			removedNodes = info.removedNodes.filter(this._isActionNode);
+			removedNodes = info.removedNodes.filter(this._isActionNode),
+			newNodes = addedNodes.filter(node => removedNodes.indexOf(node) === -1);
```

Comment:
You could switch to includes here: 
```suggestion
			newNodes = addedNodes.filter(node => !removedNodes.includes(node));
```

Reply:
:+1: